src/components/ViewSource/ViewSource.js

```diff
@@ -32,7 +32,7 @@ const ViewSource = ({
   const stripes = useStripes();
 
   const isPrintBibAvailable = !isHoldingsRecord && stripes.hasPerm('ui-quick-marc.quick-marc-editor.view');
-  const isPrintHoldingsAvailable = isHoldingsRecord && stripes.hasPerm('ui-quick-marc.quick-marc-holdings-editor.all');
+  const isPrintHoldingsAvailable = isHoldingsRecord && (stripes.hasPerm('ui-quick-marc.quick-marc-holdings-editor.all') || stripes.hasPerm('ui-quick-marc.quick-marc-holdings-editor.view'));
```

Comment:
It's ok to just check for `ui-quick-marc.quick-marc-holdings-editor.view`
Because if user has `ui-quick-marc.quick-marc-holdings-editor.all` - they will get the View permission automatically

Reply:
I think there is a logical problem with permission. When user has ui-quick-marc.quick-marc-holdings-editor.all he also get ui-quick-marc.quick-marc-holdings-editor.view perm as subperms but not opposite. When has only view permission he dosn't have ui-quick-marc.quick-marc-holdings-editor.all - and that's why isPrintHoldingsAvailable is false